internal/boxcli/pull.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2023 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package boxcli
```

Comment:
its a bit confusing to move this here but not add it as a regular command. It breaks the convention we have for the existing commands. 

Lets call this file `global_pull.go` and rename the function to `globalPullCmd()`